Dockerfile

```diff
@@ -292,7 +292,7 @@ COPY --from=generate-build /api/inspect/*.pb.go /pkg/machinery/api/inspect/
 COPY --from=go-generate /src/pkg/flannel/ /pkg/flannel/
 COPY --from=go-generate /src/pkg/imager/profile/ /pkg/imager/profile/
 COPY --from=go-generate /src/pkg/machinery/resources/ /pkg/machinery/resources/
-COPY --from=go-generate /src/pkg/machinery/config/types/v1alpha1/ /pkg/machinery/config/types/v1alpha1/
+COPY --from=go-generate /src/pkg/machinery/config/types/ /pkg/machinery/config/types/
```

Comment:
ahh I was going to say our generate was broken, since it didn't generate for the extension config ones I was working on

Reply:
No idea how I did previous configs ))